src/main/java/ch/njol/skript/expressions/ExprVillagerLevel.java

```diff
@@ -0,0 +1,114 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.expressions;
+
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.entity.Villager;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.Math2;
+import ch.njol.util.coll.CollectionUtils;
+
+@Name("Villager Level")
+@Description("Represents the current level of this villager's profession. Influences the trading options generated by the villager. " +
+	"If it is greater than their profession's maximum level, no new offers are generated. Increments when the villager fills his trading xp bar. " +
+	"Also used for badge rendering. A level must be between 1 and 5, Skript will adjust when too high/low. Villagers with a level of 2 or greater " +
+	"will keep their profession regardless if they have a work site or not. If this level is 1 and the villager has no worksite, they may loose their profession.")
+@Examples({"set {_a} to villager level of last spawned villager",
+	"set villager level of target entity to 3",
+	"add 1 to villager level of last spawned villager",
+	"remove 1 from villager level of target entity",
+	"reset villager level of last spawned villager"})
+@RequiredPlugins("Minecraft 1.14+")
+@Since("INSERT VERSION")
+public class ExprVillagerLevel extends SimplePropertyExpression<LivingEntity, Number> {
+	
+	static {
+		if (Skript.methodExists(Villager.class, "getVillagerLevel")) {
```

Comment:
Nitpick, {} here too

Reply:
I know I dont need them, I just feel in some cases it looks cleaner and more organized. 
If you really feel I should remove them I can but I see no need to remove them.

Reply:
It's more of the fact Skript has this standard of not using them if they are not needed, the conventions make mention of this but in a rather unclear manner, since one could argue it causes hard to read code.

The general gist is, if it has only one statement within the scope, remove them but still indent as if they were there. If there's multiple scopes (i.e.: an if/else if/else statement), they all need to have only one statement within their scopes for the curly braces to be avoided.

Reply:
I guess I miss read it. 
Cause I read this
```
Only when a conditional block (if or else) contains a single statement, they may be omitted
```
as "may be omitted" as in, I may omit them if I so choose. 
I will remove them and update shortly, but this should be made more clear to state "Should be omitted" not "may be omitted" to cut down on future confusion.